src/test/java/no/nav/arbeidsgiver/kandidat/kandidatsok/es/SøkAlderPrioriterteMålgrupperCvIT.kt

```diff
@@ -0,0 +1,158 @@
+package no.nav.arbeidsgiver.kandidat.kandidatsok.es
+
+import no.nav.arbeidsgiver.kandidat.kandidatsok.es.domene.EsCv
+import no.nav.arbeidsgiver.kandidat.kandidatsok.es.domene.sok.SokekriterierVeiledere
+import no.nav.arbeidsgiver.kandidat.kandidatsok.testsupport.ElasticSearchIntegrationTestExtension
+import no.nav.arbeidsgiver.kandidat.kandidatsok.testsupport.ElasticSearchTestConfiguration
+import no.nav.arbeidsgiver.kandidat.kandidatsok.testsupport.ElasticSearchTestConfiguration.DEFAULT_INDEX_NAME
+import no.nav.arbeidsgiver.kandidatsok.es.client.PrioritertMålgruppe
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZoneOffset
+import java.util.*
+
+@ExtendWith(ElasticSearchIntegrationTestExtension::class)
+class SøkAlderPrioriterteMålgrupperCvIT {
+
+    private val sokClient =
+        ElasticSearchTestConfiguration.esSokService(DEFAULT_INDEX_NAME)
+
+    private val indexerClient = ElasticSearchTestConfiguration.indexerCvService()
+
+    @BeforeEach
+    fun before() {
+        indexerClient.createIndex(DEFAULT_INDEX_NAME)
+    }
+
+    @AfterEach
+    fun after() {
+        indexerClient.deleteIndex(DEFAULT_INDEX_NAME)
+    }
+
+    @Test
+    fun KandidatSomHarFyllt50årErSenior() {
+        val cv = giveMeEsCv("100001000", LocalDate.now().minusYears(50).minusDays(1))
```

Comment:
Ikke viktig: Jeg ville brukt aktørID "anyAktørID" istedenfor "100001000" for å gjøre det tydelig at det er irrelevant for testen hvilken aktørID som brukes.